overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -37493,7 +37578,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Give me a little juice, FRIDAY!",
+          "desc:8": "This one goes fast and has §ano gradual drop§r.",
```

Comment:
A bit confusing. Maybe `Goes even faster, and has a perfectly stable Hover Mode!` (with formatting)